bin/crew

```diff
@@ -1232,7 +1237,7 @@ def resolve_dependencies
 end
 
 def install
-  if !@pkg.in_upgrade && PackageUtils.installed?(@pkg.name)
+  if !@pkg.in_upgrade && PackageUtils.installed?(@pkg.name) && !@pkg.name.start_with?('ruby_')
```

Comment:
Does this work?

Running the unit test takes 30 min... 😅

Reply:
It works in a simple local test I've got, although you might wanna double check locally before pushing a change.

Reply:
Great. I'll try to take a look later tonight...

Reply:
Ok so that doesn't work.

```ruby
Making 2 links.
Attempting removal since this is an upgrade or reinstall...
/usr/local/lib/crew/commands/remove.rb:41:in `remove': undefined method `superclass' for nil (NoMethodError)

    if @pkg.superclass.eql?('RUBY')
           ^^^^^^^^^^^
        from /usr/local/bin/crew:1283:in `install'
        from /usr/local/bin/crew:1117:in `resolve_dependencies_and_install'
        from /usr/local/bin/crew:424:in `block in upgrade'
        from /usr/local/bin/crew:416:in `each'
        from /usr/local/bin/crew:416:in `upgrade'
        from /usr/local/bin/crew:1791:in `upgrade_command'
        from /usr/local/bin/crew:1826:in `<main>'
```

Reply:
How do we have nil @pkg values? That seems like a pretty big issue